macos/FluentUI/Button/Button.swift

```diff
@@ -732,6 +839,9 @@ public struct ButtonFormat {
 
 @objc(MSFButtonColor)
 class ButtonColor: NSObject {
+	public static let firstOuterDropShadow = NSColor(named: "ButtonColors/firstOuterDropShadow", bundle: FluentUIResources.resourceBundle)
+	public static let secondOuterDropShadow = NSColor(named: "ButtonColors/secondOuterDropShadow", bundle: FluentUIResources.resourceBundle)
+	public static let innerShadow = NSColor(named: "ButtonColors/innerShadow", bundle: FluentUIResources.resourceBundle)
 	public static let brandForegroundDisabled = NSColor(named: "ButtonColors/brandForegroundDisabled", bundle: FluentUIResources.resourceBundle)
```

Comment:
Not new, but we should probably have a wrapper function in Color which returns colors from the bundle as non-optional and crashes if it doesn't exist so every caller doesn't need to handle nil